### DISCUSSION TITLE
Rocky Linux Operating System Support (install.sh)

### DISCUSSION THREAD

**Discussion by mateusfmello** [coollabsio/coolify#3676] [Feature Requests]
If the project community or @andrasbacsai  wants I can implement changes to the installation script to support Rocky Linux.

**Comment by peaklabs-dev** [comment]
Sounds amazing thank you for you contribution please open up a PR, new OS support is always welcome.

**Reply by mateusfmello** [reply]
Perfect, I'll try to do it today.

**Comment by mateusfmello** [comment]
Today I already use Coolify on Rocky Linux, but I did the entire installation manually.

**Comment by mateusfmello** [comment]
Sorry, there was a mistake on my part, I misread the part of the script that checks the operating system, it already supports Rock Linux and Alma Linux.

[install.sh#L91-L97](https://github.com/coollabsio/coolify/blob/main/scripts/install.sh#L91-L97)
